### PR TITLE
add volunteers hints

### PR DESCRIPTION
**Pull request by TMysiv**

### CODE REVIEW

File: src/pages/hint/hints/Hints.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  };
+
+  useEffect(() => {
+    getHints();
+  }, []);
+
+  return (
+    <div >
+      {user?.role === 'volunteer' &&
+      <div className={style.container}>
```
Comment (BillyAleks) [inline src/pages/hint/hints/Hints.tsx:41]: I think here something wrong with indents. Can you please check

Comment (BillyAleks) [inline src/pages/hint/hints/Hints.tsx:41]:
```suggestion
          <div className={style.container}>
```

**Review by BillyAleks — CHANGES_REQUESTED** [review]

File: src/api/hint.service.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { AxiosError } from 'axios';
+
+import { axiosInstance } from './axios-instance';
+
+class HintService {
+
```
[the remark this answers is not in the document]
Comment (TMysiv) [reply src/api/hint.service.ts:6]: fix

File: src/pages/hint/hintSingle/Hint.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+const Hint: FC = () => {
+  const { user } = useContext(CurrentUserContext)
+  const [hint, setHint] = useState<IHint>();
+  const [error, setError] = useState(null as AxiosError);
+
+  const { id } = useParams();
+
+  const getHint = async () => {
+    await hintService.getHintById(id).then(value => {
```
[the remark this answers is not in the document]
Comment (TMysiv) [reply src/pages/hint/hintSingle/Hint.tsx:19]: fix

File: src/pages/hint/hintSingle/Hint.tsx [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+      setError(error);
+    });
+  };
+
+  useEffect(() => {
+    getHint();
+  }, []);
+
+  return (
+    <div>
```
[the remark this answers is not in the document]
Comment (TMysiv) [reply src/pages/hint/hintSingle/Hint.tsx:31]: fix

File: src/pages/hint/hints/Hints.tsx (hunk 1 above)
Comment (TMysiv) [reply src/pages/hint/hints/Hints.tsx:41]: fix

Comment (TMysiv) [reply src/pages/hint/hints/Hints.tsx:41]: fix

**Review by lerik1408 — COMMENTED** [review]
Fix the conflicting files

**Pull request merged**